coins/src/adapters/utils/database.ts

```diff
@@ -138,22 +126,16 @@ async function getTokenAndRedirectDataDB(
       }))
     );
 
-    // current redirect links
-    const redirects: DbQuery[] = latestDbEntries.map((d: DbEntry) => {
-      const selectedEntries: any[] = timedDbEntries.filter(
-        (t: any) => d.PK == t.PK
-      );
-      if (selectedEntries.length == 0) {
-        return { PK: d.redirect, SK: d.SK };
-      } else {
-        return { PK: selectedEntries[0].redirect, SK: selectedEntries[0].SK };
-      }
-    });
+    // current redirects
+    const redirects: DbQuery[] = latestDbEntries
+      .map((d: DbEntry) => {
```

Comment:
I thought it'd be a good idea to start storing redirects only on SK=0 (along with decimals and symbols), meaning the redirect data is on latestDbEntries and not timedDbEntries. So I thought it'd be easier to map through latest instead of timed